tailcfg/tailcfg.go

```diff
@@ -829,6 +831,19 @@ type Hostinfo struct {
 	//       require changes to Hostinfo.Equal.
 }
 
+// VIPService represents a service created on a tailnet from the
+// perspective of a node providing that service. These services
+// have an IP address distinct from the node's IPs.
+// Ports specify which ProtoPorts are made available by this node
+// on the service's IP.
+// Active specifies whether new requests for the service should be
+// sent to this node by control.
+type VIPService struct {
+	Ports  []ProtoPortRange
```

Comment:
So this is the traffic packets we're willing to accept, but doesn't specify what we do with it, and that'll be in the serve config?

Reply:
Yep. This is the information that goes back up to control, and control doesn't need to worry about how packets are routed once they get to the client, just what the client promises to handle